examples/vanilla/disabled.html

```diff
@@ -73,7 +73,6 @@ <h1>Media Chrome Disabled-Controls Video Usage Example</h1>
           <media-time-range disabled aria-disabled="true"></media-time-range>
           <media-time-display showduration remaining disabled aria-disabled="true"></media-time-display>
           <media-captions-button default-showing disabled aria-disabled="true"></media-captions-button>
-          <media-captions-selectmenu disabled aria-disabled="true"></media-captions-selectmenu>
```

Comment:
issue(non-blocking): Realizing looking here that we don't demo disabled behavior for new menu/menu button architecture. Does that work as expected? If yes, we should probably add example(s) here.